gcp/appengine/frontend3/package.json

```diff
@@ -33,7 +33,7 @@
     "sass-loader": "12.6.0",
     "style-loader": "3.3.3",
     "webpack": "5.87.0",
-    "webpack-cli": "4.10.0",
+    "webpack-cli": "^4.10.0",
```

Comment:
I think technically this change shouldn't be necessary, and was a side-effect from you locally installing `webpack-cli`

I'm too green on how NPM works to know if this is a total no-op or whether it has any implications. Suggest reverting it unless someone with more NPM clue than me says it's benign.